Dockerfile

```diff
@@ -4,17 +4,20 @@
 #    docker build -f examples/$name/Dockerfile -t prometheus/golang-example-$name .
 
 # Builder image, where we build the example.
-FROM golang:1.9.0 AS builder
+FROM golang:1.10.3 AS builder
```

Comment:
Golang 1 has a stable and guaranteed interface, using `golang:1` would remove the need to constantly update this version with every new release.

Reply:
If that works, great.

Reply:
Thanks for the feedback. Sounds good. I've updated the PR.